pkg/resourcemanager/manager.go

```diff
@@ -2,6 +2,7 @@ package resourcemanager
 
 import (
 	"errors"
+	"k8s.io/apimachinery/pkg/types"
```

Comment:
Wanna order this with the other `k8s.io/` imports?

Reply:
i just run `go fmt` and whatever it does, i commit